src/main/java/com/sarality/db/SQLiteTable.java

```diff
@@ -169,6 +169,15 @@ public int delete(Query query) {
     return database.delete(tableDefinition.getTableName(), query.getWhereClause(), query.getWhereClauseArguments());
   }
 
+  public void execSQL(String sql, String[] queryArgs) {
```

Comment:
I thought about that and had gone in that direction first, but realized we need to pass in Context into the method for that. There might be a cleaner way to do this by preprocessing the dbfile name and db alias in the constructor, but that's a much bigger change requiring us to change the constructor of SQLiteTable.

This seemed like a reasonable compromise. Can change it later if needed since it is only exposed in SQLiteTable and not in the Table interface.